lib/url.js

```diff
@@ -1,17 +1,16 @@
-// Url
-// ===
-//
-// Url object used for tracking files in `file-list.js`
+/**
```

Comment:
Please add `'use strict'` here, it would allow to pass tests on Node 4. 

Personally I think we can remove this comment.

Reply:
Good catch! Fixed.